plugin-api.d.ts

```diff
@@ -736,6 +736,11 @@ interface PluginDataMixin {
 interface SceneNodeMixin {
   visible: boolean
   locked: boolean
+  stuckNodes: SceneNode[]
+}
+
+interface StickableMixin {
+  stuckTo?: SceneNode | null
```

Comment:
is the ? intended?

Reply:
Nope good catch! This is from my hacked version of this where I added it to every node in the widget preview docs